index.bs

```diff
@@ -279,9 +279,19 @@ added to the list of content attributes for video elements.
 The {{autoPictureInPicture}} IDL attribute MUST <a>reflect</a> the content
 attribute of the same name.
 
-The <dfn>autoPictureInPictureElement</dfn> is the video element, among all
-video elements with the {{autoPictureInPicture}} attribute currently set,
-whose {{autoPictureInPicture}} attribute was set most recently. Note that the
+All <a>documents</a> have an associated <dfn>list of Auto Picture-in-Picture
```

Comment:
It's not clear whether this means there's a single list per-document, or one list across all documents. Also, it's not clear what *appropriate* means here, is this referring to a per-document list?